### PR TITLE
WACI PEx related VC tests

### PR DESCRIPTION
**Pull request by brianorwhatever**
I've added the credential derive and verify tests for an example vax cert

### CODE REVIEW

File: packages/vc.js/src/__tests__/waci-pex/__fixtures__/credential-create.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        "https://www.w3.org/2018/credentials/v1",
+        "https://w3id.org/security/suites/bls12381-2020/v1",
+        "https://w3id.org/vaccination/v1"
+      ],
+      "id": "urn:uvci:af5vshde843jf831j128fj",
+      "type": ["VaccinationCertificate", "VerifiableCredential"],
+      "description": "COVID-19 Vaccination Certificate",
+      "name": "COVID-19 Vaccination Certificate",
+      "credentialSubject": {
+        "id": "urn:bnid:_:c14n2",
```
Comment (brianorwhatever) [inline packages/vc.js/src/__tests__/waci-pex/__fixtures__/credential-create.json:14]: this looks like the problem to me since it wasn't in the original vc

Comment (OR13) [reply packages/vc.js/src/__tests__/waci-pex/__fixtures__/credential-create.json:14]: This should be present in the derivation IF the original VC did not contain a `credentialSubject.id`...

**Comment by OR13** [comment]
I'm gonna try this https://github.com/hyperledger/aries-framework-go/blob/main/test/bbs/test/bbs.js

**Comment by OR13** [comment]
See https://github.com/w3c-ccg/ldp-bbs2020/issues/62, we can't fix this issue here.

**Pull request closed without merging**